src/ripple/protocol/impl/STAmount.cpp

```diff
@@ -1077,23 +1117,11 @@ multiply (STAmount const& v1, STAmount const& v2, Issue const& issue)
         }
     }
 
-    // Compute (numerator * denominator) / 10^14 with rounding
-    // 10^16 <= result <= 10^18
-    CBigNum v;
-
-    if ((BN_add_word64 (&v, value1) != 1) ||
-            (BN_mul_word64 (&v, value2) != 1) ||
-            (BN_div_word64 (&v, tenTo14) != 1))
-    {
-        Throw<std::runtime_error> ("internal bn error");
-    }
-
-    // 10^16 <= product <= 10^18
-    assert (BN_num_bytes (&v) <= 64);
-
-    // TODO(tom): where do 7 and 14 come from?
-    return STAmount (issue, v.getuint64 () + 7,
-        offset1 + offset2 + 14, v1.negative() != v2.negative());
+    // mantissa: (value1 * value2) / 10^14 with rounding
```

Comment:
Similar to my previous note, it would be good to expand this comment.  The comment notes the presence of the 10^14, but not *why*.  Also the two other magic numbers, 7 and 14, are skipped over altogether in the comment.